apps/backend/src/modules/users/libs/types/avatar-file.type.ts

```diff
@@ -0,0 +1,8 @@
+type AvatarFile = {
```

Comment:
This can be any other file, not only avatar. The type for File should be in files module